config/secrets.yml

```diff
@@ -11,12 +11,15 @@
 # if you're sharing your code publicly.
 
 development:
-  secret_key_base: a44a31d23f54d081b9f3737ab886048a01e2188e659d53ea608b997bb34a6dcc77b91b729a607807fb6771b6cfcd76be09bfdd0f07a6021c2c0969a07fa11755
+  secret_key_base: <%= Settings.secret.development.secret_key_base %>
 
 test:
-  secret_key_base: a44a31d23f54d081b9f3737ab886048a01e2188e659d53ea608b997bb34a6dcc77b91b729a607807fb6771b6cfcd76be09bfdd0f07a6021c2c0969a07fa11755
+  secret_key_base: <%= Settings.secret.test.secret_key_base %>
```

Comment:
Development and test secret keys should be kept in place to ease automated tests

Reply:
... except cloud service API keys.